typeql-reference/modules/ROOT/pages/index.adoc

```diff
@@ -2,7 +2,7 @@
 :page-aliases: {page-version}@reference::typeql/concepts/concept-variables.adoc, {page-version}@reference::typeql/concepts/data-instances.adoc, {page-version}@reference::typeql/concepts/overview.adoc, {page-version}@reference::typeql/overview.adoc, {page-version}@reference::typeql/concepts/types.adoc
```

Comment:
Here's the question - these are now useless right @alexjpwalker ? different component?